cmd/builder/internal/command.go

```diff
@@ -93,6 +94,7 @@ build configuration given by the "--config" argument.
 	cmd.Flags().StringVar(&cfg.Distribution.OutputPath, distributionOutputPathFlag, cfg.Distribution.OutputPath, "Where to write the resulting files")
 	cmd.Flags().StringVar(&cfg.Distribution.Go, distributionGoFlag, "", "The Go binary to use during the compilation phase. Default: go from the PATH")
 	cmd.Flags().StringVar(&cfg.Distribution.Module, distributionModuleFlag, "go.opentelemetry.io/collector/cmd/builder", "The Go module for the new distribution")
+	cmd.Flags().StringVar(&cfg.Distribution.BuildTags, "build-tags", distributionBuildTags, "A comma-separated list of build tags to consider satisfied during the build")
```

Comment:
For consistency I think we should use a const instead of the string "build-tags"